src/gui/kbperf.cpp

```diff
@@ -132,6 +133,10 @@ void KbPerf::load(CkbSettings& settings){
         }
     }
     SGroup group(settings, "Performance");
+    {
+        QString s_muteDev = settings.value("MuteDevice", "sink").toString();
+        iMuteDev = s_muteDev == "source" ? SOURCE : SINK;
```

Comment:
Please use parentheses around ternary operators.
Makes them easier to read.

Reply:
I understand that you would like to see `(s_muteDev == "source") ? SOURCE : SINK`